scan/cli.go

```diff
@@ -221,6 +221,10 @@ func createGenericAuditCmd(c *cli.Context) (*audit.GenericAuditCommand, error) {
 		auditCmd.SetWatches(strings.Split(c.String("watches"), ","))
 	}
 
+	if c.String("working-dirs") != "" {
+		auditCmd.SetWorkingDirs(strings.Split(c.String("working-dirs"), ","))
```

Comment:
Let's also trip heading and trailing white-spaces for each path, to allow users to define the `working-dirs` option as follows - 
```
--working-dirs "path/a/1, path/s/2"
```
Let's do the same for the above `--watches` option.